packages/ui/src/styles/shared-style-variables.css

```diff
@@ -564,14 +564,6 @@
     --canary-toast-icon-danger-default: var(--canary-pure-white);
     --canary-toast-icon-danger-hover: var(--canary-pure-white) / 0.65;
 
-    /* Topbar */
-    --canary-topbar-background-01: var(--canary-pure-white);
```

Comment:
Just a heads up. Its good that we are removing this. But no need, this file will be removed completely soon. So we can avoid spending time in this file.

Reply:
Sure, I understand that 🙌
I only think it would be easier to review it when it has less tokens. It took me only a few minutes to cleanup